packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -265,6 +267,8 @@ export function createResponseState(
         typeof scriptConfig === 'string' ? scriptConfig : scriptConfig.src;
       const integrity =
         typeof scriptConfig === 'string' ? undefined : scriptConfig.integrity;
+      const crossOrigin =
+        typeof scriptConfig === 'string' ? undefined : scriptConfig.crossOrigin;
```

Comment:
For crossorigin we have normalized this to `"use-credentials" | "" | null` because unless you are using use-credentials all string values behave identically to "anonymous" and "" is the shortest bytes-wise.